common/config/rush/pnpm-lock.yaml

```diff
@@ -68,7 +67,6 @@ dependencies:
   '@types/yargs': 13.0.2
   '@typescript-eslint/eslint-plugin': 2.0.0_2597c16b46d7eff3ec2c0c583ca4ae44
   '@typescript-eslint/parser': 2.0.0_eslint@6.1.0
-  abort-controller: 3.0.0
```

Comment:
Weird, I could have sworn all of this stuff got removed in my most recent PR, but apparently it came along for the ride.

Reply:
It might have been added back by another PR if they didn't rebase before re-generating the lock file.  I think it's best to think of the lockfile in terms of "eventual consistency".  As long as nothing is broken with our packages or tests, it's OK if the lockfile is a bit outdated (which it usually is since new dependencies are released continuously).